Add option for keytab already present on OE host

djasa opened the pull request:
Current aaa_sso_keytab option expects keytab to be copied on ansible
control machine. This commit adds new option aaa_sso_remote_keytab that
expects path to keytab that already exists on target machine. Remote
keytab must already have correct ownership and permissions set.

Another option is added - aaa_soo_keytab_path - which is used internally to
set the keytab path in ovirt-sso.conf. It can be also used to alter
destination for keytab copied from control machine but I decided not to
document it.

djasa committed:
Add option for keytab already present on OE host

Current aaa_sso_keytab option expects keytab to be copied on ansible
control machine. This commit adds new option aaa_sso_remote_keytab that
expects path to keytab that already exists on target machine. Remote
keytab must already have correct ownership and permissions set.

Another option is added - aaa_soo_keytab_path - which is used internally to
set the keytab path in ovirt-sso.conf. It can be also used to alter
destination for keytab copied from control machine but I decided not to
document it.

machacekondra commented on tasks/main.yml:
```diff
@@ -1,4 +1,21 @@
+- name: set up remote keytab path
+  set_fact:
+    aaa_sso_keytab_path: "{{ aaa_sso_remote_keytab }}"
+  when: aaa_sso_remote_keytab is defined
```
I am not sure I follow this code. When you just set `aaa_sso_keytab_path` to `aaa_sso_remote_keytab` always, when both are not set, but both variables  behave the same.

djasa replied:
Hi, I created this variable in order to have a simple variable expansion in sso template no matter if keytab is copied from controller or already present on remote machine. This variable is set to remote keytab path when remote keytab is used, in case of copied keytab, it's set to `/etc/httpd/httpd.keytab` (specified in defaults/main.yml).

djasa committed:
Allow use of SRV records for any profile type

Before this commit, LDAP server discovery via SRV records was only
configured for ad profiles. This commit introduces boolean
aaa_ldap_is_domain which allows to choose SRV records for other profiles
as well as use single server for ad profiles. When omitted, role
behaviour is unchanged.
